lib/index.js

```diff
@@ -67,6 +72,16 @@ module.exports = (referenceDir, config, { watch, fingerprint, sourcemaps, compac
 				runner.rerun(filepaths);
 			});
 	}
+
+	if(serve && liveserve) {
+		abort("ERROR: You can't use serve and liveserve at once");
```

Comment:
note to self: I have stylistic objections here (tone, capitalization, general consistency) - will have to elaborate some other time 📱